### ISSUE TITLE
bug: sharepoint->snowflake silent failure

### ISSUE THREAD

**Issue by yevgenypats** [cloudquery/cloudquery#15041]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Seems sharepoint plugin is working with other destinations but failure silently with snowflake

### Expected Behavior

--

### CloudQuery (redacted) config

```
spec:
  name: "sharepoint"
  registry: "github"
  path: "koltyakov/sharepoint"
  version: "v1.8.2" # provide the latest stable version

---
kind: destination
spec:
  name: "snowflake"
  path: "cloudquery/snowflake"
  registry: "cloudquery"
  version: "v3.2.11"
```

### Steps To Reproduce

-

### CloudQuery (redacted) logs

-

### CloudQuery version

-

### Additional Context

-

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by jkoppelm** [comment]
Thank you!
cloudquery version 3.26.1

Sync works fine with File, output is 99 rows, which is correct
When syncing with Snowflake, the table gets created (all columns correct), but there are now rows.
The CLI shows "99 resources" counting down.  No obvious errors in the log file:

```
2023-10-29T23:38:33Z INF batch written successfully duration=1579.6892 len=99 module=snowflake-src table=sharepoint_bpm
2023-10-29T23:38:33Z INF finished call grpc.code=OK grpc.component=server grpc.method=Write2 grpc.method_type=client_stream grpc.service=proto.Destination grpc.start_time=2023-10-29T16:38:29-07:00 grpc.time_ms=3858.341 module=cli peer.address=@ protocol=grpc
2023-10-29T23:38:33Z INF End sync destinations=["snowflake (cloudquery@cloudquery/snowflake)"] module=cli source="sharepoint (v1.8.2)" sync_time=2023-10-29T23:37:52Z
2023-10-29T23:38:33Z INF Sending sync summary to analyticsv1.cloudquery.io:443 module=cli
2023-10-29T23:38:33Z DBG sending kill signal to destination plugin module=cli
2023-10-29T23:38:33Z INF waiting for source plugin to terminate module=cli
2023-10-29T23:38:33Z INF source plugin process exited with exit status 1 module=cli
2023-10-29T23:38:33Z DBG sending kill signal to destination plugin module=cli
2023-10-29T23:38:33Z INF waiting for source plugin to terminate module=cli
2023-10-29T23:38:33Z INF source plugin process exited with exit status 1 module=cli

```

**Comment by candiduslynx** [comment]
Fixed in `v3.3.3`

**Comment by jkoppelm** [comment]
Confirmed this has been fixed.  Thank you!
